### ISSUE TITLE
Allmerge branch UI glitches

### ISSUE THREAD

**Issue by zigurana** [dissolve/PieStation#13]
I was trying to test the allmerge branch, but its difficult to use, because the UI only works 50% (mostly on the system select screen). White bg, appearing/disappearing system logos,  flickering controller art. 
This is on both the Pi and a Windows build. 
Does anybody else see this?

**Comment by dissolve** [comment]
Yes this is the main reason im not merging it in to master yet.   I think it's an issue with it hiding items it shouldn't, likely an issue of the different hide systems did not mesh correctly together

**Comment by zigurana** [comment]
Could be. Or maybe the disable systems feature, or the dynamic loading of resources from the gridview fork... 
It's difficult to debug these mega-merges. 
I think it was a bit premature to merge in de gridview stuff, as there is much activity on that at the moment. Do you maybe know of a github trick to untangle those commits from allmerge for the moment? We can reintroduce them once things settle down. 

Btw:
When in debug mode, do you also see errors on not being able to load texture this-and-that into memory? I am doubting if my freeimage libs were compiled correctly.

**Comment by zigurana** [comment]
Ok, so I've discovered that my FreeImage libraries are maybe/probably broken. I did try to compile them myself, so that would make sense...
Did anyone try to compile FreeImage using Visual Studio 2015? For ES, they need to be of the type multi-treaded (debug) dll (/MD or /MDd). 
Apparently, one needs to somehow enable plugins for freeimage to be able to parse the hardcoded resources in ES (resources\scroll_gradient.png etc). Once again, I am stumped.
